api-reference/10 UI Widgets/GridBase/1 Configuration/filterBuilderPopup.md

```diff
@@ -10,91 +10,7 @@ Configures the popup in which the integrated [filter builder](/api-reference/10%
 ---
 See the [Popup configuration](/api-reference/10%20UI%20Widgets/dxPopup '/Documentation/ApiReference/UI_Widgets/dxPopup/') for options that you can specify in this object.
 
-[note]
-
-#include widgets-config-object-option-note with {
-    optionName: "filterBuilderPopup"
-}
-
----
-##### Angular
-
-    <!-- tab: app.component.html -->
-    <dx-{widget-name} ... >
-        <dxo-filter-builder-popup
-            [onOptionChanged]="logOption">
-        </dxo-filter-builder-popup>
-    </dx-{widget-name}>
-
-    <!-- tab: app.component.ts -->
-    import { Component } from '@angular/core';
-
-    @Component({
-        selector: 'app-root',
-        templateUrl: './app.component.html',
-        styleUrls: ['./app.component.css']
-    })
-    export class AppComponent {
-        logOption(e) {
-            console.log(e.fullName + ': ' + e.value);
-        }
-    }
-
-    <!-- tab: app.module.ts -->
-    import { BrowserModule } from '@angular/platform-browser';
-    import { NgModule } from '@angular/core';
-    import { AppComponent } from './app.component';
-
-    import { Dx{WidgetName}Module } from 'devextreme-angular';
-
-    @NgModule({
-        declarations: [
-            AppComponent
-        ],
-        imports: [
-            BrowserModule,
-            Dx{WidgetName}Module
-        ],
-        providers: [ ],
-        bootstrap: [AppComponent]
-    })
-    export class AppModule { }
-
-##### Vue
-
-    <!-- tab: App.vue -->
-    <template>
-        <Dx{WidgetName} ... >
-            <DxFilterBuilderPopup
-                :on-option-changed="logOption"
-            />
-        </Dx{WidgetName}>
-    </template>
-
-    <script>
-    import 'devextreme/dist/css/dx.common.css';
-    import 'devextreme/dist/css/dx.light.css';
-
-    import Dx{WidgetName}, {
-        DxFilterBuilderPopup
-    } from 'devextreme-vue/{widget-name}';
-
-    export default {
-        components: {
-            Dx{WidgetName},
-            DxFilterBuilderPopup
-        },
-        methods: {
-            logOption(e) {
-                console.log(e.fullName + ': ' + e.value);
-            }
-        }
-    }
-    </script>
-
----
-
-[/note]
+[note]In Angular and Vue, the nested component that configures the **filterBuilderPopup** option does not support event bindings and two-way property bindings. However, you can bind to the component's events and track changes in its properties using the one-way property binding syntax.
```

Comment:
> and track changes in its properties

The only way of tracking changes in component properties I know is to use `onOptionChanged`, but we cannot use it. Are there alternatives?